modules/nf-core/tcoffee/irmsd/main.nf

```diff
@@ -0,0 +1,50 @@
+
+process TCOFFEE_IRMSD {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/t-coffee:13.46.0.919e8c6b--hfc96bf3_0':
+        'biocontainers/t-coffee:13.46.0.919e8c6b--hfc96bf3_0' }"
+
+    input:
+    tuple  val(meta),  file (msa)
+    tuple  val(meta2), file(template), file(structures)
```

Comment:
I don't see structures used anywhere, are they just read by tcoffee if they are found in the path?

Reply:
Yes! Via the template, tcoffee knows which ones to use, and looks in the same directory. 

As far as I know, this is the standard way coffee works and did not find any way to explicitly point to the structure files.